app/templates/frameworks/_framework_actions.html

```diff
@@ -42,23 +42,24 @@
       }}
     {% endif %}
   {% endif %}
-  <ul class="dm-task-list">
-    <li class="dm-task-list-item">
+  <h2 class="govuk-heading-m">Application progress</h2>
+  <ul class="dm-task-list dm-task-list__items">
+    <li class="dm-task-list__item">
       <span class="dm-task-list__task-name">
         <a href="{{ url_for('.supplier_details') }}">
           Confirm your company details
         </a>
       </span>
       {% if application_company_details_confirmed %}
-        <strong class="dm-task-list-indicator-completed" id="dm-companydetails-done">Done</strong>
+        <strong class="govuk-tag dm-task-list__tag" id="dm-companydetails-done">COMPLETED</strong>
```

Comment:
The task list example in the GOV.UK prototype kit uses title case words and leaves the CCS text transformation to make the text all-caps; can we do that here?

Reply:
We can!